python/pyarrow/tests/test_convert_builtin.py

```diff
@@ -968,25 +968,31 @@ def test_sequence_timestamp_from_int_with_unit():
     arr_s = pa.array(data, type=s)
     assert len(arr_s) == 1
     assert arr_s.type == s
-    assert repr(arr_s[0]) == "datetime.datetime(1970, 1, 1, 0, 0, 1)"
+    assert repr(arr_s[0].as_py()) == "datetime.datetime(1970, 1, 1, 0, 0, 1)"
```

Comment:
What is being tested here? If you're testing the `as_py()` result, then `repr` isn't needed. Otherwise, you may instead call `str(arr_s[0])`.

Reply:
Updated.